### PR TITLE
Post-djcelery db updates

### PR DESCRIPTION
**Pull request by efischer19**
Made following @clintonb's instructions on https://github.com/edx/devstack/pull/158

The edxapp and edxapp_csmh table updates seem correct as far as I can tell, but there look to be some demo course inserts into `ecommerce` that I'm unsure of. @clintonb, could you take a look when you get a second?

### CODE REVIEW

File: provision-lms.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@ set -o pipefail
 set -x
 
 # Load database dumps for the largest databases to save time
-./load-db.sh edxapp
-./load-db.sh edxapp_csmh
+# ./load-db.sh edxapp
```
Comment (clintonb) [inline provision-lms.sh]: Revert.

File: provision-ecommerce.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Load database dumps for the largest databases to save time
-./load-db.sh ecommerce
+# ./load-db.sh ecommerce
```
Comment (clintonb) [inline provision-ecommerce.sh]: Revert.

**Review by clintonb — COMMENTED** [review]
Demo course info is a result of a miss on my part. You should also, comment out the line in the provisioning script that loads the demo course. You may need to recreate this without those rows, as I'm not sure how Ansible will handle re-runs of the demo role.

File: provision-ecommerce.sh [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
 ./provision-ida.sh ecommerce 18130
 
 # Configure ecommerce
 docker exec -t edx.devstack.ecommerce bash -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_or_update_site --site-id=1 --site-domain=localhost:18130 --partner-code=edX --partner-name="Open edX" --lms-url-root=http://edx.devstack.lms:18000 --client-side-payment-processor=cybersource --payment-processors=cybersource,paypal --client-id=ecommerce-key --client-secret=ecommerce-secret --from-email staff@example.com'
 docker exec -t edx.devstack.ecommerce bash -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py oscar_populate_countries --initial-only'
-docker exec -t edx.devstack.ecommerce bash -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_demo_data --partner=edX'
+# docker exec -t edx.devstack.ecommerce bash -c 'source /edx/app/ecommerce/ecommerce_env && python /edx/app/ecommerce/ecommerce/manage.py create_demo_data --partner=edX'
```
Comment (efischer19) [inline provision-ecommerce.sh]: Am I correct in commenting out this line for ecommerce demo data to not be installed?

**Review by efischer19 — COMMENTED** [review]
Existing commit is a NOMERGE to allow for inspection of my environment's setup, and will be reverted.

File: provision-ecommerce.sh (hunk 3 above)
Comment (clintonb) [reply provision-ecommerce.sh]: Also https://github.com/edx/devstack/blob/master/provision-lms.sh#L28.

**Comment by efischer19** [comment]
This looks much better. For some reason, I didn't end up running migrations from scratch during provision, but I had previously checked out the commit prior to my djcelery merge in edx-platform - so it stopped there and I was able to manually fix things before bringing it up to the current tip of edx-platform's master. sql diffs looks much better from what I can tell.

@clintonb @jbarciauskas Can you give this a review-for-merge?

**Comment by efischer19** [comment]
Travis build is now making it all the way to provisioning `discovery` before error, no idea how to address that failure though. Should that db be getting dumped into github as well?

```
Traceback (most recent call last):
  File "/edx/app/discovery/discovery/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/edx/app/discovery/discovery/course_discovery/apps/course_metadata/management/commands/refresh_course_metadata.py", line 83, in handle
    if not partners:
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/models/query.py", line 260, in __bool__
    self._fetch_all()
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/models/query.py", line 1087, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql()
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 835, in execute_sql
    cursor.execute(sql, params)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 110, in execute
    return self.cursor.execute(query, args)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/cursors.py", line 250, in execute
    self.errorhandler(self, exc, value)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/connections.py", line 42, in defaulterrorhandler
    raise errorvalue
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/cursors.py", line 247, in execute
    res = self._query(query)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/cursors.py", line 411, in _query
    rowcount = self._do_query(q)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/cursors.py", line 374, in _do_query
    db.query(q)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/MySQLdb/connections.py", line 270, in query
    _mysql.connection.query(self, query)
django.db.utils.ProgrammingError: (1146, "Table 'discovery.core_partner' doesn't exist")
2017-06-21 16:51:57,152 WARNING 83 [elasticsearch] /edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/connection/base.py:82 - GET /catalog/_count [status:404 request:0.079s]
2017-06-21 16:51:57,152 WARNING 83 [elasticsearch] /edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/connection/base.py:82 - GET /catalog/_count [status:404 request:0.079s]
Traceback (most recent call last):
  File "/edx/app/discovery/discovery/manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/edx/app/discovery/discovery/course_discovery/apps/edx_haystack_extensions/management/commands/update_index.py", line 37, in handle
    record_count = self.get_record_count(backend.conn, backend.index_name)
  File "/edx/app/discovery/discovery/course_discovery/apps/edx_haystack_extensions/management/commands/update_index.py", line 24, in get_record_count
    return conn.count(index_name).get('count')
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/client/utils.py", line 69, in _wrapped
    return func(*args, params=params, **kwargs)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/client/__init__.py", line 753, in count
    doc_type, '_count'), params=params, body=body)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/transport.py", line 307, in perform_request
    status, headers, data = connection.perform_request(method, url, params, body, ignore=ignore, timeout=timeout)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/connection/http_urllib3.py", line 93, in perform_request
    self._raise_error(response.status, raw_data)
  File "/edx/app/discovery/venvs/discovery/lib/python3.5/site-packages/elasticsearch/connection/base.py", line 105, in _raise_error
    raise HTTP_EXCEPTIONS.get(status_code, TransportError)(status_code, error_message, additional_info)
elasticsearch.exceptions.NotFoundError: TransportError(404, 'IndexMissingException[[catalog] missing]')
make: *** [dev.provision.run] Error 1
```

File: provision-lms.sh [hunk 4]
```diff
@@ -6,13 +6,16 @@ set -x
 ./load-db.sh edxapp
 ./load-db.sh edxapp_csmh
 
-# Bring LMS online
+# Bring LMS and studio online
 docker-compose $DOCKER_COMPOSE_FILES up -d lms
+docker-compose $DOCKER_COMPOSE_FILES up -d studio
 
 docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && NO_PYTHON_UNINSTALL=1 paver install_prereqs'
+docker-compose exec studio bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && NO_PYTHON_UNINSTALL=1 paver install_prereqs'
```
Comment (efischer19) [inline provision-lms.sh]: I'm not 100% sure I fully understand the divide of `edxapp` being 1 thing vs `lms` and `studio` being separate things, but running a command like this made the `no module named social_django` errors go away locally.

Comment (clintonb) [reply provision-lms.sh]: Revert. The issue you're seeing is due to outdated images (relative to the code).

**Comment by efischer19** [comment]
Hooray, this passes travis. Should the latest 'Travis hack" commit be merged in, or is there a better way for us to handle the failures caused by https://blog.travis-ci.com/2017-06-21-trusty-updates-2017-Q2-launch?

See https://travis-ci.org/edx/devstack/builds/245487505 for an example failure caused by the upgrade w/o that last commit

**Comment by jbarciauskas** [comment]
I vote merge and make a PR to remove it, we can re-run that branch tomorrow or something

**Pull request merged**